src/sentry/api/helpers/constants.py

```diff
@@ -0,0 +1,2 @@
+MAX_QUERY_SUBSCTIPTIONS_HEADER = "X-Sentry-Rule-Limit"
+ALERT_RULES_COUNT_HEADER = "X-Sentry-Alert-Rule-Hits"
```

Comment:
Maybe this should be "remaining" instead of "hits" to match the API rate limits: https://docs.sentry.io/api/ratelimits/#headers

Reply:
Eh I could see it either way, up to you

Reply:
In this case, I don't think we should rename this because it already existed on the `combined-rules` api. i just also wanted to move this so it's consistent between alert-rules and combined-rules.